Ical.Net.Tests/GetOccurrenceTests.cs

```diff
@@ -152,7 +152,7 @@ public void EnumerationChangedException()
     }
 
     [Test]
-    public void GetOccurrencesShouldEnumerate()
+    public void GetOccurrencesWithRecurrenceIdShouldEnumerate()
```

Comment:
Also works when `RECURRENCE-ID;VALUE=DATE` (it case this is a useful test, too)
`´`C#
    [Test]
    public void GetOccurrencesWithRecurrenceId_DateOnly_ShouldEnumerate()
    {
        const string ical = @"BEGIN:VCALENDAR
PRODID:-//github.com/ical-org/ical.net//NONSGML ical.net 5.0//EN
VERSION:2.0
BEGIN:VEVENT
UID:789012
DTSTART;VALUE=DATE:20231001
DTEND;VALUE=DATE:20231002
RRULE:FREQ=MONTHLY;BYMONTHDAY=1
SUMMARY:Monthly Report Due
END:VEVENT
BEGIN:VEVENT
UID:789012
RECURRENCE-ID;VALUE=DATE:20231101
DTSTART;VALUE=DATE:20231115
DTEND;VALUE=DATE:20231116
SUMMARY:Monthly Report Due (Rescheduled)
END:VEVENT
END:VCALENDAR
";

        var collection = Calendar.Load(ical);
        var startCheck = new DateTime(2023, 10 ,1);
        var occurrences = collection.GetOccurrences<CalendarEvent>(startCheck, startCheck.AddMonths(1))
            .OrderBy(x => x)
            .ToList();

        CalDateTime[] expectedStartDates = [
            new CalDateTime(2023, 10, 1),
            new CalDateTime(2023, 11, 15), // the replaced occurrence
            new CalDateTime(2023, 12,1)
        ];

        // For DTEND=20231002
        Assert.That(occurrences.Select(x => x.Period.StartTime), Is.EqualTo(expectedStartDates.Take(1)));

        // For DTEND=20231231
        var occurrences2 = collection.GetOccurrences<CalendarEvent>(new CalDateTime(startCheck), new CalDateTime(2023, 12, 31))
            .OrderBy(x => x)
            .ToList();

        Assert.That(occurrences2.Select(x => x.Period.StartTime), Is.EqualTo(expectedStartDates.Take(3)));
    }
```

Reply:
Great, thank you for the test!